baseplate/integration/trace_headers.py

```diff
@@ -0,0 +1,122 @@
+import re
+
+
+LOWER_HEX_16 = re.compile(r"\A[0-9a-f]{16}\Z")
+LOWER_HEX_32 = re.compile(r"\A[0-9a-f]{32}\Z")
+
+LOWER_HEX_16_BYTES = re.compile(b"\A[0-9a-f]{16}\Z")
+LOWER_HEX_32_BYTES = re.compile(b"\A[0-9a-f]{32}\Z")
+
+
+class TraceHTTPHeaders(object):
+    """Helper class for trace headers in an HTTP call.
```

Comment:
If this is it for the docstring, I think you can fit the closing `"""` on the same line.